packages/bentley/breakdown-trees/package.json

```diff
@@ -57,7 +57,7 @@
     "@bentley/ui-ninezone": "^2.19.9",
     "@bentley/webgl-compatibility": "^2.19.9",
     "@bentley/webpack-tools": "1.14.1",
-    "@bentley/measure-tools-react": "workspace:*",
+    "@bentley/measure-tools-react": "^0.5.0",
```

Comment:
I guess this is needed because we don't have the 2.x version in there repo anymore?

Reply:
correct, will revert this once i add the 2.x ver in a separate pr